### PR TITLE
[MONIT-28837] Fix Incorrect AWS SQS queue names

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: proxy/src/main/java/com/wavefront/agent/queueing/SQSQueueFactoryImpl.java [hunk 1]
```diff
@@ -112,14 +121,13 @@ private String getOrCreateQueue(String queueName) {
...
-            addAttributesEntry(QueueAttributeName.VisibilityTimeout.toString(), "60").
-            setQueueName(queueName);
+        request
+            .addAttributesEntry(QueueAttributeName.MessageRetentionPeriod.toString(), "1209600")
+            .addAttributesEntry(QueueAttributeName.ReceiveMessageWaitTimeSeconds.toString(), "20")
+            .addAttributesEntry(QueueAttributeName.VisibilityTimeout.toString(), "60")
+            .setQueueName(queueName);
         CreateQueueResult result = client.createQueue(request);
         queueUrl = result.getQueueUrl();
-        queues.put(queueName, queueUrl);
```
Comment (joannak-vmware) [inline proxy/src/main/java/com/wavefront/agent/queueing/SQSQueueFactoryImpl.java:122]: Why are we removing this?

**Review by joannak-vmware — APPROVED** [review]
Address comment, LGTM otherwise

File: proxy/src/main/java/com/wavefront/agent/queueing/SQSQueueFactoryImpl.java (hunk 1 above)
Comment (laullon) [reply proxy/src/main/java/com/wavefront/agent/queueing/SQSQueueFactoryImpl.java:122]: That's done on the line 76, doing it here can cause concurrent execution.
` String queueUrl = queues.computeIfAbsent(queueName, x -> getOrCreateQueue(queueName));`

Comment (joannak-vmware) [reply proxy/src/main/java/com/wavefront/agent/queueing/SQSQueueFactoryImpl.java:122]: Thanks, LGTM

**Review by sbhakta-vmware — CHANGES_REQUESTED** [review]
lgtm

**Pull request merged**